R/plot_dot.R

```diff
@@ -759,7 +760,7 @@ get_construct_type <- function(model, construct) {
       }
     }
   }
-
+  #cat(paste(construct_type, ":", construct, "\n"))
```

Comment:
There seem to be debug statements commented out in multiple files -- can we remove them?